README.md

```diff
@@ -49,21 +33,22 @@ For now, the best way to run Livebook is by cloning it and running it locally:
 
 You will need [Elixir v1.11](https://elixir-lang.org/install.html) or later.
 
-We will work on other distribution modes (escripts, Docker images, etc) once
-we start distributing official releases.
+We will work on other distribution modes (escripts, Docker images, etc) once we start distributing official releases.
 
-## Security considerations
+### Security considerations
 
-Livebook is built to document and execute code. Anyone with access to a
-Livebook instance will be able to access any file and execute any code
-in the machine Livebook is running.
+Livebook is built to document and execute code. Anyone with access to a Livebook instance will be able to access any file and execute any code in the machine Livebook is running.
 
-For this reason, `Livebook` only binds to the 127.0.0.1, allowing access
-to happen only within the current machine. When running `Livebook` in the
-production environment - the recommended environment - we also generate a
-token on initialization and we only allow access to the Livebook if said
-token is supplied as part of the URL.
+For this reason, `Livebook` only binds to the 127.0.0.1, allowing access to happen only within the current machine. When running `Livebook` in the production environment - the recommended environment - we also generate a token on initialization and we only allow access to the Livebook if said token is supplied as part of the URL.
 
+### Environment variables
+<!-- Environment variables -->
+
+The following environment variables configure Livebook:
+
+  * LIVEBOOK_SECRET_KEY_BASE - this sets a secret key that is used to sign and encrypt payloads use by Livebook. By default a random one is generated at boot but you can set your own with this environment variable. The secret key base must be at least 64 characters long and it can be generated via commands such as: `openssl rand -base64 48`
```

Comment:
```suggestion
  * LIVEBOOK_SECRET_KEY_BASE - this sets a secret key that is used to sign and encrypt payloads used by Livebook. By default a random one is generated at boot but you can set your own with this environment variable. The secret key base must be at least 64 characters long and it can be generated via commands such as: `openssl rand -base64 48`
```